static/app/components/replays/breadcrumbs/breadcrumbItem.tsx

```diff
@@ -201,6 +202,43 @@ function CrumbErrorIssue({frame}: {frame: ErrorFrame}) {
   );
 }
 
+function HTMLTree({description, frame}: {description: string; frame: ClickFrame}) {
+  const location = useLocation();
+  const organization = useOrganization();
+
+  const componentName =
+    isClickFrame(frame) && frame.data.node?.attributes['data-sentry-component'];
+  const lastComponentIndex =
+    description.lastIndexOf('>') === -1 ? 0 : description.lastIndexOf('>') + 2;
+
+  return (
+    <Description title={description} showOnlyOnOverflow isHoverable>
+      {componentName ? (
+        <Fragment>
+          <div style={{display: 'inline'}}>
+            {description?.toString().substring(0, lastComponentIndex)}
```

Comment:
do we need the `toString()` here if we pass it in as a string up above?

Reply:
oh thanks for catching that!